composer.json

```diff
@@ -22,15 +22,15 @@
         }
     ],
     "require": {
-        "php":                               ">=5.3.3",
-        "zendframework/zend-mvc":            "2.*",
-        "zendframework/zend-eventmanager":   "2.*",
-        "zendframework/zend-stdlib":         "2.*",
-        "zendframework/zend-servicemanager": "2.*",
-        "zendframework/zend-modulemanager":  "2.*",
-        "zendframework/zend-version":        "2.*",
-        "zendframework/zend-debug":          "2.*",
-        "zendframework/zend-view":           "2.*"
+        "php":                               "^5.5 || ^7.0",
+        "zendframework/zend-mvc":            "^2.7",
```

Comment:
use `"^2.7 || ^3.0.0@dev"` in favor of https://github.com/zendframework/ZendSkeletonApplication/pull/329 to avoid this error:
```
$ composer update
You are running composer with xdebug enabled. This has a major impact on runtime performance. See https://getcomposer.org/xdebug
Loading composer repositories with package information
Updating dependencies (including require-dev)                             
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Can only install one of: zendframework/zend-mvc[2.7.x-dev, 3.0.x-dev].
    - Can only install one of: zendframework/zend-mvc[2.7.x-dev, 3.0.x-dev].
    - zendframework/zend-developer-tools dev-zf3 requires zendframework/zend-mvc ^2.7 -> satisfiable by zendframework/zend-mvc[2.7.x-dev].
    - Installation request for zendframework/zend-developer-tools dev-zf3 -> satisfiable by zendframework/zend-developer-tools[dev-zf3].
    - Installation request for zendframework/zend-mvc ^3.0.0@dev -> satisfiable by zendframework/zend-mvc[3.0.x-dev].
```